pallets/eth-bridge/src/benchmarking.rs

```diff
@@ -120,10 +130,12 @@ benchmarks! {
         let active_tx = ActiveTransaction::<T>::get().expect("is active");
         let msg_hash_string = active_tx.msg_hash.to_string();
         // TODO: We need a real signature as this benchmark fails at present
```

Comment:
```suggestion
```

Reply:
I think this comment is still relevant because the real benchmark will fail due to this signature. Will fix this in a new PR.